xpk.py

```diff
@@ -1522,8 +1522,8 @@ def install_kueue_on_cluster(args) -> int:
     0 if successful and 1 otherwise.
   """
   command = (
-      'kubectl apply -f'
-      ' https://github.com/kubernetes-sigs/kueue/releases/download/v0.4.1/manifests.yaml'
+      'kubectl apply --server-side --force-conflicts -f'
```

Comment:
`--force-conflicst` needed to upgrade from non-server-side preconfigured gke clusters